tsconfig.settings.json

```diff
@@ -0,0 +1,13 @@
+{
+    "compilerOptions": {
+      "module": "CommonJS",
+      "sourceMap": true,
+      "target": "ES2018",
```

Comment:
Why ES2018 here? More generically, we should decide how wide of browser range we need to support. I'd love to use ES2020 to use `bigint`, but for Gitcoin it feels more appropriate to target something older for wider device/browser support

Reply:
whatever you suggest we can do, just let us know. otherwise i'll leave at es2018.

Reply:
Created an issue to track this here: https://github.com/dcgtc/dcgrants/issues/3

Let's leave at ES2018 for this PR though.